source/lib/thunderbird.js

```diff
@@ -64,6 +74,71 @@ function showNewMessageNotification(message, isRSS) {
         text: text,
         iconURL: utils.getIcon(),
     });
+
+}
+
+function format(message, format, callback){
+
```

Comment:
I found this way to separate sender name from sender email, which seems better than using regex.